src/main/java/com/netflix/imflibrary/st0377/HeaderPartition.java

```diff
@@ -1344,6 +1347,7 @@ public enum EssenceTypeEnum {
         VisuallyImpairedTextEssence(Composition.SequenceTypeEnum.VisuallyImpairedTextSequence),
         CommentaryEssence(Composition.SequenceTypeEnum.CommentarySequence),
         KaraokeEssence(Composition.SequenceTypeEnum.CommentarySequence),
+        ForcedNarrativeSequence(Composition.SequenceTypeEnum.ForcedNarrativeSequence),
```

Comment:
The enum entry should be `ForcedNarrativeEssence` instead of `ForcedNarrativeSequence`.
```suggestion
        ForcedNarrativeEssence(Composition.SequenceTypeEnum.ForcedNarrativeSequence),
```